inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -140,6 +140,11 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     float threshold;
     InferenceEngine::CNNNetwork cnnNetwork;
     std::shared_ptr<InferenceEngine::Core> core;
+    ngraph::PartialShape inputDynamicShape;
```

Comment:
```suggestion
    std::vector<ngraph::PartialShape> inputDynamicShape;
    std::vector<ngraph::Shape> targetStaticShape;
    std::vector<std::vector<ngraph::Shape>> targetStaticShapes;
```

I believe we need to store `std::vector` inputDynamicShape, targetStaticShape and targetStaticShapes for tests which have more that one input. It's necessary for example in `EltwiseLayerTest`

Reply:
I will try this after rebase and update the PR.